lib/publiccloud/basetest.pm

```diff
@@ -78,6 +78,11 @@ sub _cleanup {
     eval { $self->cleanup(); } or bmwqemu::fctwarn($@);
 
     $self->{provider} //= [];
+    my $flags = $self->test_flags();
+    if ($flags->{publiccloud_multi_module} &&
```

Comment:
#8483  introducing `PUBLIC_CLOUD_ALWAYS_CLEANUP` and this is somehow about the same right ? 
@pdostal , @cfconrad , @ldevulder - I think we should check a bigger picture if this futures not overlap or duplicate each other . Also maybe we can come up with single solution which would fit all sides ?

Reply:
Sounds like yes. We have to check what is the best method: adding a variable like what I did or this solution (with a change in the flag name maybe, e.g. `cleanup_if_fail` for example).

Reply:
I am sorry but I don't know the background of this. In my case, I want the test continue running and do the cleaning always at the end. This so far works for me.

Reply:
I clarified with @ldevulder